.storybook/config.js

```diff
@@ -5,9 +5,14 @@ import { configure, addDecorator } from '@storybook/react';
 import { COLORS } from '../src/constants';
 import Wrapper from './components/Wrapper';
 
-import '../src/global-styles';
+import GlobalStyles from '../src/global-styles';
 
-const WrapperDecorator = storyFn => <Wrapper>{storyFn()}</Wrapper>;
+const WrapperDecorator = storyFn => (
+  <>
+    <GlobalStyles />
+    <Wrapper>{storyFn()}</Wrapper>
+  </>

```

Comment:
Nit: We've been using `Fragment` instead of `<>` in this project:

```js
import React, { Fragment } from 'react';

<Fragment>
  <GlobalStyles />
  <Wrapper>{storyFn()}</Wrapper>
</Fragment>
```

It does exactly the same thing, but it's a bit more google-able, so for folks who haven't seen it before, it's not as mysterious/magical.